websites/D/Discohook/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Discohook",
+  "author": {
+    "id": "150987570715688960",
+    "name": "Ninja#4321"
```

The discriminator is a mistake in the `pmd` command.
```suggestion
    "name": "Ninja"
```

---

No problem, discriminator removed.